docs/manpages/rec_control.1.md

```diff
@@ -43,9 +43,23 @@ To dump the cache to disk, execute:
      respond. Set to 0 for infinite.
 
 # COMMANDS
+add-nta *DOMAIN* [*REASON*]
+:    Add a Negative Trust Anchor for *DOMAIN*, suffixed optionally with *REASON*.
+
+add-ta *DOMAIN* *DSRECORD*
+:    Add a Trust Anchor for *DOMAIN* with DS record data *DSRECORD*.
+
 current-queries
 :    Shows the currently active queries.
 
+clear-nta [*DOMAIN*]...
+:    Remove Negative Trust Anchor for one or more *DOMAIN*s. If *DOMAIN* is
+     empty, remove all configured Negative Trust Anchors.
```

Comment:
'empty' or 'no domain specified'? I think we might want to be explicit here, use '*' or something. If somebody automates setting/clearing of NTAs it's pretty easy to accidentally pass an empty list